.circleci/config.yml

```diff
@@ -64,6 +65,7 @@ jobs:
           scan-all-projects: true
           skip-unresolved: false
           os: macos
+          additional-arguments: --prune-repeated-subdependencies
```

Comment:
The dependency tree is way too big for Snyk to handle for Android builds without passing in this argument. Newer versions of Expo create a lot of modules and snyk eventually gives up the security scan unless we pass in this argument. This catches vulns, but doesn't give all the paths for reaching a specific vuln lib. But that's simple enough to do locally, so no big loss.